### PR TITLE
MM-22013 - Allow users to specify default download locations

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/desktop/pull/1383

Updated:
- User's Guide > Getting Started > Using Mattermost Desktop App > App Options
   - Created a new subsection called "Download Location" and documented how to change where files are saved to the local machine

### CODE REVIEW

**Review by esethna — APPROVED** [review]

**Comment by esethna** [comment]
Probably don't need a dev review on end user docs unless there's a technical change to review

File: source/help/apps/desktop-guide.rst
Comment (justinegeffen) [inline source/help/apps/desktop-guide.rst:52]:
```suggestion
3. Select **Remove**.
```

File: source/help/apps/desktop-guide.rst
Comment (justinegeffen) [inline source/help/apps/desktop-guide.rst:50]:
```suggestion
1. On Windows, go to **... > File > Settings**. On macOS or Linux, go to **Mattermost > Preferences**.
```

File: source/help/apps/desktop-guide.rst
Comment (justinegeffen) [inline source/help/apps/desktop-guide.rst:40]:
```suggestion
1. On Windows, go to **... > File > Settings**. On macOS or Linux, go to **Mattermost > Preferences**.
```

File: source/help/apps/desktop-guide.rst
Comment (justinegeffen) [inline source/help/apps/desktop-guide.rst:59]:
```suggestion
The options appear on the Settings page, available on Windows from **... > File > Settings**, and on macOS or Linux from **Mattermost > Preferences**.
```

File: source/help/apps/desktop-guide.rst
Comment (justinegeffen) [inline source/help/apps/desktop-guide.rst:93]:
```suggestion
1. On Windows, go to **... > File > Settings**. On macOS or Linux, go to **Mattermost > Preferences**.
```

**Review by justinegeffen — APPROVED** [review]
Some minor suggestions for naming but otherwise LGTM and approving to unblock.

**Comment by amyblais** [comment]
@cwarnermm There are some merge conflicts here, can you help resolve and then we can merge this?

**Comment by amyblais** [comment]
@cwarnermm Looks like there are merge conflicts again after I merged https://github.com/mattermost/docs/pull/4568. Sorry :(

**Pull request merged**
